backend/task.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+from sklearn.cluster import KMeans
+import pandas as pd
+import numpy as np
+import re
+import nltk
+
+from gensim.models import Word2Vec
+from time import time  # To time our operations
+from collections import defaultdict  # For word frequency
+import logging  # Setting up the loggings to monitor gensim
+logging.basicConfig(format="%(levelname)s - %(asctime)s: %(message)s", datefmt= '%H:%M:%S', level=logging.INFO)
+
+import matplotlib.pyplot as plt
+
+data = pd.read_csv("floop_data_15k.csv")
+
+data.shape
+
+
+#To remove special characters and punctuation from our dataset
+from string import punctuation
+
+punctuations = punctuation
+
+def solution(sentence):
+    for p in punctuations:
+        sentence = sentence.replace(p, '')
+    return sentence
+
+x = data["Field1"].apply(solution)
+pattern = "[^a-zA-Z0-9]"
+x_cleaned = [re.sub(pattern," ",text) for text in x]
+
+x_lowered = [text.lower() for text in x_cleaned]
+x_lowered
+
+x_lowered[0]
+
+nltk.download('punkt')
+
+x_tokenized = [nltk.word_tokenize(text) for text in x_lowered]
+
+nltk.download('wordnet')
+from nltk.stem import WordNetLemmatizer
+lemma = WordNetLemmatizer()
+
+x_lemmatized = [[lemma.lemmatize(word) for word in text] for text in x_tokenized]
+
+print(x_lemmatized[0])
+
+
+
+
+
+# For classification data whether good or bad.
+
+w2v_model = Word2Vec(min_count=20,
+                     window=2,
+                     size=300,
+                     sample=6e-5, 
+                     alpha=0.03, 
+                     min_alpha=0.0007, 
+                     negative=20,
+                     workers= 1 )
+
+t = time()
+
+w2v_model.build_vocab(x_lemmatized, progress_per=10000)
+
+print('Time to build vocab: {} mins'.format(round((time() - t) / 60, 2)))
+
+t = time()
+
+w2v_model.train(x_lemmatized, total_examples=w2v_model.corpus_count, epochs=30, report_delay=1)
+
+print('Time to train the model: {} mins'.format(round((time() - t) / 60, 2)))
+
+w2v_model.init_sims(replace=True)
+
+w2v_model.save("word2vec11.model")
+
+word_vectors = Word2Vec.load("/content/word2vec11.model").wv
```

Comment:
This should be `word_vectors = Word2Vec.load("word2vec11.model").wv`. Otherwise you get a file not found error.

Reply:
Yes, It was removed, I returned back.

Reply:
No, you shouldn't add the file back. You just need to change where the model is being loaded from. 

When I ran your script, the model was saved to `/backend` but the script then tried to load it from `/backend/content`, which caused an error. I fixed it by changing the load path to just `"word2vec11.model"`.

Reply:
I returned back the file and updated.

Reply:
The file `backend/word2vec11.model` should be deleted. You can add it to .gitignore to avoid adding it accidentally. Also, please see [this comment](https://github.com/North-Seattle-College/ad440-winter2022-tuesday-repo/pull/30#discussion_r796279028).